.github/workflows/on-trigger.yml

```
@@ -62,6 +62,10 @@ jobs:
     if: ${{ github.event_name == 'push' && github.ref_type == 'branch' && contains(fromJSON('["develop", "release"]'), github.ref_name) }}
     uses: ./.github/workflows/reusable-check-missing-commits.yml
 
+  check-single-commit:
+    if: ${{ github.event_name == 'merge_group' }}
+    uses: ./.github/workflows/reusable-check-commit-message.yml
+
```

Will this workflow ever run under a `merge_group` trigger?

---

You're right. I thought this workflow would get triggered by merge groups, but it won't. I can adjust this check.

---

> You're right. I thought this workflow would get triggered by merge groups, but it won't. I can adjust this check.

The only downside of changing it to `push` is that once it fires, the damage it done. Plus further pushes could hide the failure.

Should this check also be added to `on-pr.yml`? For the `merge_group`?

---

Yes, I got things upside-down. Let me push another change to this PR.